react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import logo from './logo.svg';
+import './App.css';
+
+
+function CustomButton() {
+  return (
+    <div>
+      <button
+        style={{ backgroundColor: "red" }}>
+        I
+      </button>
+      <button
+        style={{ backgroundColor: "green" }}>
+        know
+      </button>
+      <button
+        style={{ backgroundColor: "lightblue" }}>
+        React
+      </button>
+    </div>
+  )
+}
+
+function App() {
+  return (
+  <CustomButton />
```

Comment:
We want to be calling all instances of the CustomButton component here by passing in props.